### PR TITLE
Describe the nabc syntax.

### PR DESCRIPTION
**Pull request by jakubjelinek**
I've finally found time to describe the nabc syntax in the documentation, including an commented example.

### CODE REVIEW

File: doc/GregorioNabcRef.tex [hunk 1]
```diff
@@ -89,6 +89,246 @@
...
+descriptor}s, optionally followed by a sequence of \textit{significant letter
+descriptor}s.
+
+The \textit{horizontal spacing adjustment descriptor}
+consists of a sequence of horizontal spacing adjustment characters
+\texttt{/} and \texttt{`}.
+\begin{itemize}
+  \item \texttt{//} move by \texttt{nabclargerspace} skip to the right
+  \item \texttt{/} move by \texttt{nabcinterelementspace} skip to the right
+  \item \texttt{``} move by \texttt{nabclargerspace} skip to the right
```
Comment (henryso) [inline doc/GregorioNabcRef.tex]: How is this different from // ?

File: doc/GregorioNabcRef.tex [hunk 2]
```diff
@@ -89,6 +89,246 @@
...
+characters optionally followed by a number:
+\begin{itemize}
+  \item \texttt{S} modification of the mark
+  \item \texttt{G} modification of the grouping (neumatic break)
+  \item \texttt{M} melodic modification
+  \item \texttt{-} addition of episema
+  \item \texttt{>} augmentive liquescence
+  \item ~~\texttt{\~} diminutive liquescence
+\end{itemize}
+If the Dom Cardine's table contains multiple glyphs with the same modifiers,
```
Comment (henryso) [inline doc/GregorioNabcRef.tex]: You should drop "the" before "Dom".

File: doc/GregorioNabcRef.tex [hunk 3]
```diff
@@ -89,6 +89,246 @@
...
+\begin{itemize}
+  \item \texttt{t} \sneume{ta} tractulus
+  \item \texttt{u} \sneume{ta-} tractulus with episema
+  \item \texttt{v} \sneume{suv1} tractulus with double episema
+  \item \texttt{w} \sneume{gr} gravis
+  \item \texttt{x} \sneume{st>} liquescens stropha
+  \item \texttt{y} \sneume{gr-} gravis with episema
+\end{itemize}
+Only subpunctis are normally used in neume classification, the prepunctis is
+a made up concept to describe the rising sequence of punctis, tractulis etc.
```
Comment (henryso) [inline doc/GregorioNabcRef.tex]: Rather than "a made up", I'd say "an nabc".

File: doc/GregorioNabcRef.tex (hunk 1 above)
Comment (henryso) [inline doc/GregorioNabcRef.tex]: I see from the example below that the backquote is a skip to the left.  You should probably correct that here and the next line.

**Comment by henryso** [comment]
Maybe another person should give this an extra once-over, but it looks good to me.

**Comment by eschwab** [comment]
I'll try to proof read this tomorrow.

File: doc/GregorioNabcRef.tex [hunk 4]
```diff
@@ -89,6 +89,246 @@
 }
 
 {
+\setlength\parindent{0cm}
+
+The \texttt{nabc} language allows to describe some adiastematic neumes, for
```
Comment (eschwab) [inline doc/GregorioNabcRef.tex]: 'allows to describe' would read better as 'provides the ability to describe'

File: doc/GregorioNabcRef.tex [hunk 5]
```diff
@@ -89,6 +89,246 @@
...
+
+\begin{verbatim}
+nabc-lines: 1;
+(f3) AL(ef~|ta>)le(fg/hggf|peclhgpi)lú(ef~|ta>)ia.(f.|ta-) (,)
+(ii//|bv-|gh!ivHG//|vi-hhppu2su1sut1|fhg/|to|eef.|pt) (;)
+\end{verbatim}
+
+The \texttt{nabc} snippets are then separated by \texttt{|} character from
+\texttt{gabc} snippets.  Every \texttt{gabc} snippet may be followed by
+up to \texttt{nabc-lines} \texttt{nabc} snippets, followed optionally
```
Comment (eschwab) [inline doc/GregorioNabcRef.tex]: This sentence is slightly confusing. May be better as:

Every \texttt{gabc} snippet may be followed by multiple \texttt{nabc} snippets. The maximum number of \texttt{nabc} snippets is the number declared in the header field \texttt{nabc-lines: x;}. More \texttt{gabc} snippets can follow a \texttt{nabc} snippet group in an alternating pattern. (gabc|nabc|gabc|nabc|...etc...)

File: doc/GregorioNabcRef.tex [hunk 6]
```diff
@@ -89,6 +89,246 @@
...
+The \textit{pitch descriptor} allows to specify the vertical position of the
+neume.  There are no staves, so the vertical position is only rough.  For
+pitches the same letters as in \texttt{gabc} are used, \texttt{a} through
+\texttt{m}.  If the \textit{pitch descriptor} is missing, the default is
+\texttt{hf}, otherwise it consists of the letter \texttt{h} followed by
+the pitch letter.  Within the \textit{complex glyph descriptor}, each
+\textit{basic glyph descriptor} has its own pitch, but in the current fonts
+there are no glyphs with different relative pitches, so if you use a
+\textit{pitch descriptor} on any of the \textit{basic glyph descriptor}s
+in the \textit{complex glyph descriptor}, better use the same one on all the other
```
Comment (eschwab) [inline doc/GregorioNabcRef.tex]: 'better use' --> 'it is best to use'

**Comment by eschwab** [comment]
Looks good to me.

**Pull request merged**